src/fidgets/farcaster/Feed.tsx

```diff
@@ -78,19 +86,44 @@ const Feed: React.FC<FidgetArgs<feedFidgetSettings>> = ({
   // https://tanstack.com/query/latest/docs/framework/react/reference/useInfiniteQuery
   // https://www.radix-ui.com/primitives/docs/components/scroll-area
 
+  const [ref, inView] = useInView();
+
+  useEffect(() => {
+    if (inView && hasNextPage) {
```

Comment:
`if (inView && hasNextPage && !isFetchingNextPage)` to prevent repeated requests